.github/workflows/nightly.yml

```diff
@@ -27,7 +27,17 @@ jobs:
 
       - uses: Swatinem/rust-cache@v1
 
-      - name: Compile
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Compile frontend
+        run: |
+          cd site/frontend
+          npm ci
```

Comment:
Spell it out as `clean-install`? (I had to look up with `ci` meant.)

Reply:
I think that for people that use `npm` often it might be the other way around (I actually had no idea that it's called `clean-install`, it's colloquially referred as npm ci => you run that in CI xD).

Reply:
We use this in multiple places (Dockerfile also), so I'll change it everywhere at once in a different PR.